views/homepage/base.html

```diff
@@ -0,0 +1,64 @@
+{% set locale = localeInfo.lang %}
+{% set localeDir = localeInfo.locale %}
+<!DOCTYPE html>
+<html lang="{{ locale }}" dir="{{ localeInfo.direction }}">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+    <meta name="bramble-host" content="{{editorHOST}}">
+
+    <meta name="twitter:card" content="summary_large_image">
+    <meta name="twitter:site" content="@mozlearn">
+    <meta name="twitter:creator" content="@mozlearn">
+    <meta name="twitter:title" content="{{ gettext("thimbleHeading") | striptags }}">
+    <meta name="twitter:description" content="{{ gettext("thimbleDescription") | striptags }}">
+    <meta name="twitter:image" content="{{ APP_HOSTNAME }}/img/social.png">
+
+    <meta property="og:url" content="{{ APP_HOSTNAME }}/{{ locale }}/">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{{ gettext("thimbleHeading") | striptags }}">
+    <meta property="og:description" content="{{ gettext("thimbleDescription") | striptags }}">
+    <meta property="og:image" content="{{ APP_HOSTNAME }}/img/social.png">
+    <meta property="og:image:width" content="1200">
+    <meta property="og:image:height" content="630">
+
+    <link rel="stylesheet" href="/resources/stylesheets/userbar.css">
+    <link rel="stylesheet" href="/homepage/stylesheets/style.css">
+    <link rel="stylesheet" href="/homepage/stylesheets/get-involved.css">
+    <link rel="stylesheet" href="/homepage/stylesheets/gallery.css">
+    <link rel="stylesheet" href="/homepage/stylesheets/features.css">
+
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Arvo" rel="stylesheet">
```

Comment:
Unfortunately, this won’t look good in many locales (we had the same issue for the copyright petition: https://github.com/MozillaFoundation/Advocacy/issues/775 for reference)
And it will be worse with Zilla since it does only support English for now.